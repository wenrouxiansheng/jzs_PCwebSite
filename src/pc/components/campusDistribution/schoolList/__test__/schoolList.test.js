import React from "react";
import renderer from "react-test-renderer";
import { shallow } from "enzyme";
import SchoolList from "../index";

describe("PC:<SchoolList />", () => {
  it("render", () => {
    const wrapper = renderer.create(<SchoolList />);
    let tree = wrapper.toJSON();

    //渲染测试
    expect(tree).toMatchSnapshot();
  });
  it("paging event", () => {
    const NUM = 2;
    const wrapper = shallow(<SchoolList />);

    wrapper.find("Pagination").simulate("change", NUM);
    expect(wrapper.find("Pagination").prop("current")).toBe(NUM);
  });

});
