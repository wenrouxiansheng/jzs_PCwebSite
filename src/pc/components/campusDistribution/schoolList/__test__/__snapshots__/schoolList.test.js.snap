// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PC:<SchoolList /> render 1`] = `
<div
  className="schoolList"
  style={Object {}}
>
  <div
    className="animationTitle"
    style={Object {}}
  >
    <h3>
      <b>
        校区分布
      </b>
    </h3>
    <h6>
      Campus Distribution
    </h6>
  </div>
  <div
    className="list"
  >
    <div>
      <img
        alt=""
      />
      <div
        className="container"
      >
        <h1>
          通州校区
        </h1>
        <div>
          <span>
            开设课程：
          </span>
          <p>
            成人/少儿舞蹈培训班、成人/少儿美术培训班、学唱歌培训班、吉他/尤克里里/架子鼓/钢琴培训班、艺考考级培训班等；
          </p>
        </div>
        <div>
          <span>
            开设班级：
          </span>
          <p>
            1v1精品小班、1v2精品小班、1v3精品小班、系统集体大班、艺考考级冲刺班、艺考考级模拟班等多种班型。
          </p>
        </div>
        <p
          className="position"
        >
          <img
            alt=""
            src="test-file-stub"
          />
          北京市通州万达广场C区101-109
        </p>
        <p
          className="phone"
        >
          <img
            alt=""
            src="test-file-stub"
          />
          400-900-8898
        </p>
        <button
          className="btnsh_udesk_im"
        >
          立即咨询
        </button>
      </div>
    </div>
    <div>
      <img
        alt=""
      />
      <div
        className="container"
      >
        <h1>
          崇文门校区
        </h1>
        <div>
          <span>
            开设课程：
          </span>
          <p>
            成人/少儿舞蹈培训班、成人/少儿美术培训班、学唱歌培训班、吉他/尤克里里/架子鼓/钢琴培训班等；
          </p>
        </div>
        <div>
          <span>
            开设班级：
          </span>
          <p>
            1v1精品小班、1v2精品小班、1v3精品小班、系统集体大班等多种班型。
          </p>
        </div>
        <p
          className="position"
        >
          <img
            alt=""
            src="test-file-stub"
          />
          北京市东城区崇文门外大街18号便宜坊大厦
        </p>
        <p
          className="phone"
        >
          <img
            alt=""
            src="test-file-stub"
          />
          400-900-8898
        </p>
        <button
          className="btnsh_udesk_im"
        >
          立即咨询
        </button>
      </div>
    </div>
    <div>
      <img
        alt=""
      />
      <div
        className="container"
      >
        <h1>
          中关村校区
        </h1>
        <div>
          <span>
            开设课程：
          </span>
          <p>
            成人/少儿舞蹈培训班、成人/少儿美术培训班、学唱歌培训班、吉他/尤克里里/架子鼓/钢琴培训班等；
          </p>
        </div>
        <div>
          <span>
            开设班级：
          </span>
          <p>
            1v1精品小班、1v2精品小班、1v3精品小班、系统集体大班等多种班型。
          </p>
        </div>
        <p
          className="position"
        >
          <img
            alt=""
            src="test-file-stub"
          />
          北京市海淀区知春路113号银网中心
        </p>
        <p
          className="phone"
        >
          <img
            alt=""
            src="test-file-stub"
          />
          400-900-8898
        </p>
        <button
          className="btnsh_udesk_im"
        >
          立即咨询
        </button>
      </div>
    </div>
    <div>
      <img
        alt=""
      />
      <div
        className="container"
      >
        <h1>
          亚运村校区
        </h1>
        <div>
          <span>
            开设课程：
          </span>
          <p>
            成人/少儿舞蹈培训班、成人/少儿美术培训班、学唱歌培训班、吉他/尤克里里/架子鼓/钢琴培训班等；
          </p>
        </div>
        <div>
          <span>
            开设班级：
          </span>
          <p>
            1v1精品小班、1v2精品小班、1v3精品小班、系统集体大班等多种班型。
          </p>
        </div>
        <p
          className="position"
        >
          <img
            alt=""
            src="test-file-stub"
          />
          北京市朝阳区慧忠里103号洛克时代中心
        </p>
        <p
          className="phone"
        >
          <img
            alt=""
            src="test-file-stub"
          />
          400-900-8898
        </p>
        <button
          className="btnsh_udesk_im"
        >
          立即咨询
        </button>
      </div>
    </div>
    <div>
      <img
        alt=""
      />
      <div
        className="container"
      >
        <h1>
          公主坟校区
        </h1>
        <div>
          <span>
            开设课程：
          </span>
          <p>
            成人/少儿舞蹈培训班、成人/少儿美术培训班、学唱歌培训班、吉他/尤克里里/架子鼓/钢琴培训班等；
          </p>
        </div>
        <div>
          <span>
            开设班级：
          </span>
          <p>
            1v1精品小班、1v2精品小班、1v3精品小班、系统集体大班等多种班型。
          </p>
        </div>
        <p
          className="position"
        >
          <img
            alt=""
            src="test-file-stub"
          />
          北京市海淀区万寿路大诚中心
        </p>
        <p
          className="phone"
        >
          <img
            alt=""
            src="test-file-stub"
          />
          400-900-8898
        </p>
        <button
          className="btnsh_udesk_im"
        >
          立即咨询
        </button>
      </div>
    </div>
  </div>
  <div
    className="pagination"
  >
    <ul
      className="ant-pagination"
      style={Object {}}
      unselectable="unselectable"
    >
      <li
        aria-disabled={true}
        className="ant-pagination-prev ant-pagination-disabled"
        onClick={[Function]}
        onKeyPress={[Function]}
        tabIndex={null}
        title="Previous Page"
      >
        <button
          className="ant-pagination-item-link"
          disabled={true}
          tabIndex={-1}
          type="button"
        >
          <span
            aria-label="left"
            className="anticon anticon-left"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="left"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M724 218.3V141c0-6.7-7.7-10.4-12.9-6.3L260.3 486.8a31.86 31.86 0 000 50.3l450.8 352.1c5.3 4.1 12.9.4 12.9-6.3v-77.3c0-4.9-2.3-9.6-6.1-12.6l-360-281 360-281.1c3.8-3 6.1-7.7 6.1-12.6z"
              />
            </svg>
          </span>
        </button>
      </li>
      <li
        className="ant-pagination-item ant-pagination-item-1 ant-pagination-item-active"
        onClick={[Function]}
        onKeyPress={[Function]}
        tabIndex="0"
        title={1}
      >
        <a
          rel="nofollow"
        >
          1
        </a>
      </li>
      <li
        className="ant-pagination-item ant-pagination-item-2"
        onClick={[Function]}
        onKeyPress={[Function]}
        tabIndex="0"
        title={2}
      >
        <a
          rel="nofollow"
        >
          2
        </a>
      </li>
      <li
        className="ant-pagination-item ant-pagination-item-3"
        onClick={[Function]}
        onKeyPress={[Function]}
        tabIndex="0"
        title={3}
      >
        <a
          rel="nofollow"
        >
          3
        </a>
      </li>
      <li
        className="ant-pagination-item ant-pagination-item-4"
        onClick={[Function]}
        onKeyPress={[Function]}
        tabIndex="0"
        title={4}
      >
        <a
          rel="nofollow"
        >
          4
        </a>
      </li>
      <li
        aria-disabled={false}
        className="ant-pagination-next"
        onClick={[Function]}
        onKeyPress={[Function]}
        tabIndex={0}
        title="Next Page"
      >
        <button
          className="ant-pagination-item-link"
          disabled={false}
          tabIndex={-1}
          type="button"
        >
          <span
            aria-label="right"
            className="anticon anticon-right"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="right"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M765.7 486.8L314.9 134.7A7.97 7.97 0 00302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 000-50.4z"
              />
            </svg>
          </span>
        </button>
      </li>
      <li
        className="ant-pagination-options"
      >
        <div
          className="ant-pagination-options-quick-jumper"
        >
          Go to
          <input
            onBlur={[Function]}
            onChange={[Function]}
            onKeyUp={[Function]}
            type="text"
            value=""
          />
          
        </div>
      </li>
    </ul>
  </div>
</div>
`;
